contracts/test/TestPaymasterConfigurableMisbehavior.sol

```diff
@@ -124,6 +128,13 @@ contract TestPaymasterConfigurableMisbehavior is TestPaymasterEverythingAccepted
             return IPaymaster.GasAndDataLimits(limits.acceptanceBudget * 9, limits.preRelayedCallGasLimit, limits.postRelayedCallGasLimit,
             limits.calldataSizeLimit);
         }
+        if (preHigherThanAcceptance) {
+            return IPaymaster.GasAndDataLimits(
+                limits.acceptanceBudget,
+                limits.preRelayedCallGasLimit + limits.acceptanceBudget + relayHub.calldataGasCost(limits.calldataSizeLimit),
```

Comment:
too much magic in contract instead of in the test itself.

Reply:
Fixing, though this is a test contract, so does it matter where this magic happens?